test-scenarios.md

```diff
@@ -0,0 +1,139 @@
+MLS Interop Test Scenarios
+==========================
```

Comment:
So these are essentially scripts the test runner would execute among the different implementations?

Reply:
Yep, that's the idea.